### DISCUSSION TITLE
I think I found a bug from Osu! Lazer

### DISCUSSION THREAD

**Discussion by blackheart04044** [ppy/osu#24536] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Particle when Kiai active
Spin too fast
Voice when select music

### Screenshots or videos

https://drive.google.com/drive/folders/1vaDx77vHLTai2hSwqsVl_CbMbqLq2bfW?usp=sharing

### Version

Lazer 2023-812-0

### Logs

[database.log](https://github.com/ppy/osu/files/12329977/database.log)
[input.log](https://github.com/ppy/osu/files/12329978/input.log)
[network.log](https://github.com/ppy/osu/files/12329982/network.log)
[performance.log](https://github.com/ppy/osu/files/12329983/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12329984/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12329985/updater.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24503
https://github.com/ppy/osu/issues/24519
